admin-dev/themes/default/js/tree.js

```diff
@@ -213,13 +210,13 @@ Tree.prototype =
 
 			$.get(
 				'ajax-tab.php',
-				{controller:'AdminProducts',token:currentToken,action:'getCategoryTree',type:idTree,fullTree:1,selected:selected, inputName:name,useCheckBox:useCheckBox},
+				{ controller:'AdminProducts', token:currentToken, action:'getCategoryTree', type:idTree, fullTree:1, selected:selected, inputName:name, useCheckBox:useCheckBox },
 				function(content)
 				{
 					$('#' + idTree).html(content);
```

Comment:
can be improved by putting `$('#' + idTree)` in a variable. But it's not blocking. Thanks @crezzur ;)

Reply:
Added a new commit where the `$('#' + idTree)` is stored in the variable called  `targetTree`